test/extensions/wasm/wasm_test.cc

```diff
@@ -175,10 +175,10 @@ TEST_P(WasmTest, EmscriptenVersion) {
   uint32_t major = 9, minor = 9, abi_major = 9, abi_minor = 9;
   EXPECT_TRUE(wasm->getEmscriptenVersion(&major, &minor, &abi_major, &abi_minor));
   EXPECT_EQ(major, 0);
-  EXPECT_LE(minor, 2);
+  EXPECT_LE(minor, 3);
```

Comment:
Could you add support for the 0.3 metadata version (i.e. extract the new metadata value)?

Reply:
Done.